packages/components/src/renderer/location.tsx

```diff
@@ -0,0 +1,82 @@
+import { Status, Wrapper } from '@googlemaps/react-wrapper'
+import React, { useEffect, useRef } from 'react'
+import { MessageTypeHandlerProps } from '../typings'
+
+export const MapContainer: React.FC<MessageTypeHandlerProps<'location'>> = ({ latitude, longitude, address }) => {
+  const ref = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (!ref.current) {
+      return
+    }
+
+    const position = { lat: latitude, lng: longitude }
+
+    const map = new window.google.maps.Map(ref.current, {
+      center: position,
+      zoom: 14
+    })
+
+    const marker = new google.maps.Marker({
+      position,
+      map,
+      title: address
+    })
+
+    return () => {
+      marker.setMap(null)
+    }
+  })
+
+  return <div ref={ref} id="map" />
+}
+
+const Fallback = ({ latitude, longitude }: { latitude: number; longitude: number }): React.ReactElement => {
+  const link = `https://www.google.com/maps/search/?api=1&query=${latitude},${longitude}`
+  return (
+    <p>
+      Location:&nbsp;
+      <a href={link} target="_blank">
```

Comment:
I wouldn't put Location: as a fallback. It's not translatable and also it doesn't align with what other channels do. Twilio, whatsapp and others for example puts the title and adresse and the link afterwards.